Products/CMFPlone/controlpanel/browser/maintenance.pt

```diff
@@ -105,6 +105,10 @@
                             tal:condition="form_name"
                             tal:content="form_name">Form name</legend>
 
+                    <p>
+                        <strong i18n:translate="text_zope_database_name">Database name:</strong> <span tal:replace="view/dbName" />
```

Comment:
Where the dbName comes from does not matter. It is part of a translatable string so it should get an i18n name. The lines after it, with the current database size, have an i18n:name too.